catalog/task/cleanup-workspace/0.2/tests/test-cleanup-workspace.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: test-cleanup-workspace
+spec:
+  description: |
+    Run the cleanup-workspace task with a custom workspace
+    and verify that the cleanup is performed in the given subdirectory.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: input
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: input
+        steps:
+          - name: put-content-in-workspace
+            image: quay.io/hacbs-release/release-utils:5b1a1cd9fd68625cab5573ce62e0d87e6f93f341
+            script: |
+              #!/usr/bin/env sh
+              
+              INPUT_PATH="$(workspaces.input.path)"
+              mkdir -p $INPUT_PATH/dir-to-be-cleaned-up/
+              echo "This file and dir will be deleted" >> $INPUT_PATH/dir-to-be-cleaned-up/cleanup_test.txt
+              mkdir -p $INPUT_PATH/dir-to-be-kept/
+              echo "Test successful" >> $INPUT_PATH/dir-to-be-kept/cleanup_check.txt
+              ls -R $INPUT_PATH/
```

Comment:
this is probably nice as you write the task, but I don't think it needs to be in the final version. We should have some faith that `echo foo >> file` works